support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateSalesforceContact.scala

```diff
@@ -32,20 +35,123 @@ class CreateSalesforceContact extends ServicesHandler[CreateSalesforceContactSta
     }
   }
 
-  private def getCreateZuoraSubscriptionState(
-    state: CreateSalesforceContactState,
-    response: SalesforceContactRecords
-  ): CreateZuoraSubscriptionState =
-    CreateZuoraSubscriptionState(
-      state.requestId,
-      state.user,
-      state.giftRecipient,
-      state.product,
-      state.analyticsInfo,
-      state.paymentMethod,
-      state.firstDeliveryDate,
-      state.promoCode,
-      response,
-      state.acquisitionData
-    )
+}
+class NextState(state: CreateSalesforceContactState) {
```

Comment:
this is all temporary code to turn the "old style" state into the product specific state.  Once CreateSalesforceContactState is converted, it will move upstream again.